### PR TITLE
Update tracing samples

### LINKED ISSUE #10453: Tracing OpenTelemetry  Sample code is wrong
1
Section : 
[Link](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md)
image

Suggestion: 
Change code as follow:
image

2
Section : 
[Link](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-tracing-opentelemetry/src/samples/CreateConfigurationSettingTracingSample.md)
image

Suggestion: 
Change code as follow:
image

3
Section : 
[Link](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-tracing-opentelemetry/src/samples/ListKeyVaultSecretsTracingSample.md)
image

Suggestion: 
Change code as follow:
image

4
Section : 
[Link](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core-tracing-opentelemetry/src/samples/PublishEventsTracingSample.md)
image

Suggestion: 
Change code as follow:
image

### PR DESCRIPTION
**Pull request by samvaity**
Fixes #10453

### CODE REVIEW

File: sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md [hunk 1]
```diff
@@ -18,97 +18,83 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package
...
 <dependency>
     <groupId>com.azure</groupId>
     <artifactId>azure-core-tracing-opentelemetry</artifactId>
     <version>1.0.0-beta.4</version>
 </dependency>
 ```
-[//]: # ({x-version-update-end})
 
 #### Sample demonstrates tracing when asynchronously creating and listing secrets from a Key Vault using [azure-security-keyvault-secrets][azure_keyvault_secrets] client library.
 ```java
```
Comment (alzimmermsft) [inline sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md:41]: We should update this Sample to use an inject snippet, should be able to include an entire file in a snippet

Comment (samvaity) [reply sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md:41]: Yup, planning to do that in #10321 and use these sample snippets in the Readme content.

File: sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md [hunk 2]
```diff
@@ -18,97 +18,83 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package
 </dependency>
 ```
 
-[//]: # ({x-version-update-start;com.azure:azure-security-keyvault-secrets;current})
 ```xml
+<!-- SDK dependencies   -->
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.0.5</version>
```
Comment (joshfree) [inline sdk/core/azure-core-tracing-opentelemetry/src/samples/AsyncListKeyVaultSecretsSample.md:26]: nit: might as well make this 1.0.6 now

File: sdk/core/azure-core-tracing-opentelemetry/src/samples/ListKeyVaultSecretsTracingSample.md [hunk 3]
```diff
@@ -18,83 +18,72 @@ Sample uses **[opentelemetry-sdk][opentelemetry_sdk]** as implementation package
 </dependency>
 ```
 
-[//]: # ({x-version-update-start;com.azure:azure-security-keyvault-secrets;current})
 ```xml
+<!-- SDK dependencies   -->
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.0.5</version>
```
Comment (joshfree) [inline sdk/core/azure-core-tracing-opentelemetry/src/samples/ListKeyVaultSecretsTracingSample.md:26]: 1.0.6

**Pull request merged**